lib/src/native/rtc_video_renderer_impl.dart

```diff
@@ -34,6 +36,9 @@ class RTCVideoRendererNative extends VideoRenderer {
   @override
   MediaStream? get srcObject => _srcObject;
 
+  @override
+  Function? onResize;
```

Comment:
I think we can be more clear about the signature of onResize at the interface level.
void Function()? onResize;

Reply:
done 
https://github.com/flutter-webrtc/webrtc-interface/commit/159fa75f35ee2ce12360c27eaae2df8910a9e943